index.d.ts

```diff
@@ -0,0 +1,34 @@
+/// <reference types="node"/>
+import {Options as GravatarUrlOptions} from 'gravatar-url';
+
+declare namespace getGravatar {
+	type Options = GravatarUrlOptions;
+}
+
+/**
+Get a Gravatar image.
+
+@param email - Email matching a Gravatar profile.
+
+@example
+```
+import {promisify} from 'util';
+import * as fs from 'fs';
+import getGravatar = require('get-gravatar');
+
+const writeFileP = promisify(fs.writeFile);
```

Comment:
We can now start using `fs.promises` in readme examples. Example: https://github.com/sindresorhus/junk/commit/a7c280daddca13fe5d2f731235b97bdfb5940fda